spec/npm/npmSpec.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+
+describe('The npm module', function() {
+	let testEnv;
+
+	beforeEach(function() {
+		testEnv = this.getTestEnv();
+	});
+
+	it('should install and run simply named packages', async function() {
+		const scriptSource = `
+			const uniqueCount = npm.dedupe([1, 2, 2, 3]).length;
+			here.file('output').make(true).content = uniqueCount;
+		`;
+
+		await testEnv.runLemonScript(scriptSource);
+
+		const outputPath = testEnv.nativePathFor('output');
+		let value = parseInt(fs.readFileSync(outputPath));
+		expect(value).toBe(3);
+	});
+
+	it('should install and run scoped packages', async function() {
+		const scriptSource = `
+			const octokit = new npm['@octokit/rest'].Octokit();
+
+			let lemon = (await octokit.rest.repos.listForUser({ username: "Cykelero" }))
+				.data.find(({ name }) => name == 'tasklemon');
+
+			here.file('output.json').make(true).content = JSON.stringify(lemon);
+		`;
+
+		await testEnv.runLemonScript(scriptSource);
+
+		const outputPath = testEnv.nativePathFor('output.json');
+		let value = JSON.parse(fs.readFileSync(outputPath));
+		expect(value.full_name).toBe('Cykelero/tasklemon');
+	});
+
+	it('should recognize pinned versions for scoped packages', async function() {
+		// The #require must be in column 0 of the script, thus the unusual formatting
+		const scriptSource = `#require @octokit/core@3.0.0
```

Comment:
Just in case, let's test a more specific version, like `3.2.4`, to catch any issues with parsing the minor and patch version numbers.

Reply:
Implemented by 963b109